package.py

```diff
@@ -825,7 +825,7 @@ def _zip_write_with_filter(zip_stream, path_filter, source_path, prefix,
 
 
 @contextmanager
-def install_pip_requirements(query, requirements_file):
+def install_pip_requirements(query, requirements_file, tmp_dir=None):
```

Comment:
I believe having a default `None` value is not needed as `install_pip_requirements()` is [called from a single place.](https://github.com/terraform-aws-modules/terraform-aws-lambda/pull/230/files#diff-34b2a63c4b18a635042766329eca28d8801699eecf5e93eaedaeda1309931381L788)

Reply:
I was just being safe, but I can remove if needed.

Reply:
@artificial-aidan Please do.